pkgdb2/api/acls.py

```diff
@@ -143,7 +143,7 @@ def api_acl_update():
             SESSION.commit()
             output['output'] = 'ok'
             output['messages'] = messages
-        except pkgdblib.PkgdbException, err:
+        except pkgdblib.exceptions.PkgdbException, err:
```

Comment:
I wonder if it wouldn't be better to add a ``from pkgdb.lib.exceptions import PkgdbException`` at the top and use this